Apps/Examples/ExamplesUITests/ExamplesUITests.swift

```diff
@@ -69,3 +41,20 @@ final class ExamplesUITests: XCTestCase {
         }
     }
 }
+
+extension XCTestCase {
+    /// Query SpringBoard for alerts and wait for `Allow` or `Allow While Using App` button
+    func acceptLocationPermissionAlert(timeout: TimeInterval) {
+        let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
```

Comment:
can we make use this this https://developer.apple.com/documentation/xctest/xctestcase/1496273-adduiinterruptionmonitor? or did it have some problem in the past?